src/common/utils/model.ts

```diff
@@ -74,6 +74,12 @@ export function fetchAllCountyProjections(snapshotUrl: string | null = null) {
         summaryWithTimeseries =>
           getStateName(summaryWithTimeseries.state) !== undefined,
       )
+      .filter(summaryWithTimeseries => {
+        // We don't want to return county projections for counties that we
+        // don't have in the regions instance
+        const region = regions.findByFipsCode(summaryWithTimeseries.fips);
+        return region instanceof County;
+      })
       .map(summaryWithTimeseries => {
         const region = regions.findByFipsCode(summaryWithTimeseries.fips);
         assert(region, 'Failed to find region ' + summaryWithTimeseries.fips);
```

Comment:
Since you're making sure we have a region, you can call `regions.findByFipsCodeStrict` above